### PR TITLE
mrc-3847: Add privacy notice linked from login page

### PR DESCRIPTION
**Pull request by r-ash**
## Description

This PR will
* Add a Privacy statement page
* Add links from landing page to about, news, privacy statement and accessibility statement

## Type of version change

Minor

## Checklist

- [x] I have incremented version number, or version needs no increment
- [x] The build passed successfully, or failed because of ADR tests

### CODE REVIEW

File: src/app/static/src/app/store/translations/locales.ts [hunk 1]
```diff
@@ -517,6 +541,42 @@ const en: Translations = {
...
+        "                 <li>tell you about your privacy rights;</li>\n" +
+        "                 <li>tell you how the law protects you;</li>\n",
+    privacyPreambleBody2: "This policy is based on the <a href=\"https://www.imperial.ac.uk/about-the-site/privacy/\" target='_blank' rel='noopener noreferrer'>Imperial College privacy policy</a>.",
+    privacyCollectHeading: "What do we collect?",
+    privacyCollectBody: "If you use this application we will ask you to provide:",
+    privacyCollectList: "<li>Your email address</li>" +
+        "                <li>Input data required for the model to produce estimates including Spectrum PJNZ, country geojson, population data, survey data, ART data and ANC data</li>" +
+        "                <li>Parameters for running a model fit</li>",
+    privacyThirdPartyHeading: "Information we collect from third parties",
+    privacyThirdPartyList: "<li>If you login to Naomi using the HIV Tools single sign-on service you direct the service to send us profile information. Your email address will be stored on the Naomi servers.</li>" +
```
Comment (EmmaLRussell) [inline src/app/static/src/app/store/translations/locales.ts:556]: Perhaps single sign-on should be capitalised as it is on the login page.. "Log in with your HIV Tools Single Sign-On account?

File: src/app/static/src/app/store/translations/locales.ts [hunk 2]
```diff
@@ -517,6 +541,42 @@ const en: Translations = {
...
+        "                 <li>We will not share your information with any other organisation for marketing, market research, or any other commercial purposes.</li>",
+    privacyRightsHeading: "Your rights",
+    privacyRightsList: "<li>You can find out what information we hold about you by submitting a subject access request to <a href=\"mailto:naomi-support@imperial.ac.uk\">naomi-support@imperial.ac.uk</a> and ask us not to use the information we collect.</li>" +
+        "               <li>You also have the right to have data erased if it is no longer necessary for the purpose for which it was originally collected/processed, or if there are no overriding legitimate grounds for the processing. This is sometimes known as 'the right to be forgotten'.</li>" +
+        "               <li>To find out more, please see - <a href=\"https://www.imperial.ac.uk/admin-services/secretariat/information-governance/data-protection/internal-guidance/guide-10/\">Guide 12 - Individual Rights | Administration and support services | Imperial College London</a></li>",
```
Comment (EmmaLRussell) [inline src/app/static/src/app/store/translations/locales.ts:577]: For consistency, I think this should open in another tab (as the other link to non-Naomi Imperial page does).

**Review by EmmaLRussell — COMMENTED** [review]
Looks good! I wonder if the privacy statement should also appear in the 'Online support' menu, as accessibility does.

File: src/app/static/templates/login.ftl [hunk 3]
```diff
@@ -28,6 +28,12 @@
     </script>
 </head>
 <body>
+    <div class="links text-sm-right">
+        <a class="btn" href="https://reside-ic.github.io/projects/naomi/" target="_blank">About</a>
+        <a class="btn" href="https://naomi.unaids.org/news" target="_blank">News</a>
+        <a class="btn" href="/privacy">Privacy</a>
+        <a class="btn" href="/accessibility">Accessibility</a>
+    </div>
```
Comment (LekanAnni) [inline src/app/static/templates/login.ftl:36]: I don't think this looks aesthetically appealing. Can we put this links somewhere else ? Can we have them as footer ? 

Something like this 
Screenshot 2023-02-06 at 11 08 18

**Review by LekanAnni — CHANGES_REQUESTED** [review]

File: src/app/static/templates/login.ftl (hunk 3 above)
Comment (r-ash) [reply src/app/static/templates/login.ftl:36]: Yeah my first thought was to put them here but then got caught up with trying to make it look good next to all the logos, let me give it another go

**Comment by r-ash** [comment]
Ok had a bit of a go at tidying this up, let me know what you think. I think still some improvements we can make but I feel like I've spent long enough on this.

This is the page with SSO login
image

And with standard login
image

File: src/app/static/src/app/components/header/DataExplorationSupportMenu.vue [hunk 4]
```diff
@@ -3,8 +3,14 @@
         <drop-down text="support" :right="true" :delay="true" style="flex: none">
             <a class="dropdown-item"
                @click="toggleErrorReportModal"
+               tabindex="0"
```
Comment (r-ash) [inline src/app/static/src/app/components/header/DataExplorationSupportMenu.vue:6]: Noticed this item wasn't styled consistently with other buttons in the drop down. This shows as black text. Setting `tabindex` fixes this, copied over from main page support menu.

**Review by LekanAnni — COMMENTED** [review]
Looks great, 

As discussed, a few modifications for different screens.

Bootstraps will automatically resize content based base on client screens but because you have written a lot of css then you will have to support different screens. 

To improve responsiveness, try to adapt your css into bootstraps. see some pointers here 
* https://getbootstrap.com/docs/4.0/utilities/flex/
* https://getbootstrap.com/docs/5.0/components/buttons/
* https://getbootstrap.com/docs/5.0/utilities/spacing/


Distorted view on my MacBook

Screenshot 2023-02-23 at 13 23 28

**Review by EmmaLRussell — APPROVED** [review]
Looks good to me - failing test is ADR. 

We've got quite far from the addition of the privacy notice in the discussions of the login page styling, so perhaps styling tidy-up could be done in a separate ticket?

I wonder if the issue of the partner logos wrapping could be remedied somewhat by pulling those out of the container class div? That should allow the logos to take up the whole width of the screen.

**Review by LekanAnni — APPROVED** [review]
Looks perfect 👍 

Thanks for making the changes

**Pull request merged**